routes/results/results.controller.js

```diff
@@ -1,17 +1,23 @@
 const { routeUtils, getSessionData } = require('./../../utils')
 const { Schema } = require('./schema.js')
-const { getBenefits } = require('./getBenefits');
+const { getBenefits } = require('./getBenefits')
+const _ = require('lodash')
 
 const getData = (req) => {
   /**
    * If there's querystring data use it,
    * otherwise get it from the session.
    */
-  if (req.query) {
-    return req.query;
+  console.log(`queryString ${JSON.stringify(req.query)}`)
+  if (req.query === undefined || _.isEmpty(req.query)) {
+    return getSessionData(req);
+  }
+  try {
+    return JSON.parse(Buffer.from(req.query.q, 'base64').toString())
+  } catch (err) {
+    req.locals.logError(`Thrown error: ${JSON.stringify(err)} Invalid QueryString ${JSON.stringify(req.query)}`)
```

Comment:
I think this is probably supposed to be `req.locals.log` ... when I trigger an error, I get an error message about this:

![image](https://user-images.githubusercontent.com/1187115/81178483-43862880-8f76-11ea-85dd-23414ed3381e.png)

see:
![image](https://user-images.githubusercontent.com/1187115/81179248-6402b280-8f77-11ea-8221-02085bb27a70.png)

Reply:
😬 Good catch, I originally had multiple loggers in there but removed them,

Reply:
Fixed